end-project.js

```diff
@@ -45,6 +48,19 @@ module.exports = (context, options) => {
         [require.resolve('@babel/plugin-proposal-class-properties'), { loose: true }]
     );
 
+    config.plugins.push(['@babel/plugin-transform-runtime', {
+        // We already have preset-env injecting the core-js polyfills automatically
+        corejs: false,
+        // Do not replace babel helpers with calls to moduleName
+        helpers: false,
+        // Automatically requires @babel/runtime/regenerator and avoid
```

Comment:
This should be `Add import calls whenever the `regeneratorRuntime` global is found`